static/sass/_lagunita.scss

```diff
@@ -344,6 +344,18 @@ section.introduction h1.sr {
     min-height: 100px;
 }
 
+// Modal styling
+#seq_content .vert-mod {
+    section.modal.staff-modal,
+    section.modal.history-modal {
+        left: 0 !important;
+        top: 0 !important;
+        width: 90% !important;
+        height: 85% !important;
+        margin: 5% !important;
```

Comment:
![What is so important](https://www.reviveourhearts.com/media/filer_public/59/5e/595e660c-b0a2-4e7e-90fc-3d531a2788b0/blogtwimportsomuchtodo.jpg)